### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by ReenSami**

### CODE REVIEW

**Comment by shymonk** [comment]
I'm afraid can not merge your PR before PEP8 issues are fixed.

**Comment by ReenSami** [comment]
I didn't update DB that's why lines are commented, You can look into changed files.
I need imageColumn so add it in columns and its working fine for me.

**Comment by ReenSami** [comment]
PEP8 issues are fixed now.

File: table/columns/imagecolumn [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+from django.utils.safestring import mark_safe
+from django.utils.html import escape
+from django.template import Template, Context
```
Comment (shymonk) [inline table/columns/imagecolumn]: leave blank

File: table/columns/imagecolumn [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from table.columns import Column
+from table.utils import Accessor
+
+
+class ImageColumn(Column):
+    def __init__(self, field=None, image_title=None, *args, **kwargs):
+        kwargs["sortable"] = False
+        kwargs["searchable"] = False
+        self.image_path = field
```
Comment (shymonk) [inline table/columns/imagecolumn]: `self.image_path` seems useless.

File: table/columns/imagecolumn [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        self.image_title = image_title
+        super(ImageColumn, self).__init__(field=field, *args, **kwargs)
+
```
Comment (shymonk) [inline table/columns/imagecolumn]:
```
def render(self, obj):
    path = Accessor(self.field).resolve(obj)
    if isinstance(self.image_title, Accessor):
        title = self.image_title.resolve(self.obj)
    else:
        title = self.image_title
    
    template = Template('{%% load static %%}<img src="{%% static "%s" %%}" title="%s">' % (path, title))
    return template.render(Context())
```

Comment (shymonk) [inline table/columns/imagecolumn]: This will simplify the logic.

**Pull request merged**